ush/radmon_verf_time.sh

```diff
@@ -0,0 +1,574 @@
+#
+#
+# Usage:  radmon_verf_time.sh PDATE
+#
+#   Input script positional parameters:
+#     PDATE		processing date
+#  			yyyymmddcc format; required 
+#
+#   Imported Shell Variables:
+#     DO_DATA_RPT	switch to build the data report
+#                       defaults to 1 (on)
+#     RADMON_SUFFIX	data source suffix
+#                       defauls to opr
+#     EXECradmon        executable directory
+#                       defaults to current directory 
+#     FIXgdas           fixed data directory
+#                       defaults to current directory
+#     RAD_AREA          global or regional flag
+#                       defaults to global
+#     TANKverf_rad	data repository
+#                       defaults to current directory 
+#     SATYPE		list of satellite/instrument sources
+#        		defaults to none
+#     MAIL_TO		email recipients
+#			defaults to none
+#     MAIL_CC		email cc recipients
+#			defaults to none
+#     VERBOSE           Verbose flag (YES or NO)
+#                       defaults to NO
+#     LITTLE_ENDIAN     flag for little endian machine
+#                       defaults to 0 (big endian)
+#     USE_ANL		use analysis files as inputs in addition to 
+#                         the ges files.  Default is 0 (ges only)
+#
+#   Modules and files referenced:
+#     scripts    : 
+#
+#     programs   : $NCP
+#                  $time_exec
+#
+#     fixed data : gdas_radmon_base.tar
+#
+#     input data : $data_file
+#                  
+#     output data: $time_file
+#                  $time_ctl
+#                  $pgmout
+#                  $bad_pen
+#                  $bad_chan
+#                  $report
+#                  $diag_report
+#                   
+#
+# Remarks:
+#
+#   Condition codes
+#      0 - no problem encountered
+#     >0 - some problem encountered
+#
+####################################################################
+
+#  Command line arguments.
+export PDATE=${1:-${PDATE:?}}
+
+# Directories
+FIXgdas=${FIXgdas:-$(pwd)}
+EXECradmon=${EXECradmon:-$(pwd)}
+TANKverf_rad=${TANKverf_rad:-$(pwd)}
+
+# File names
+#pgmout=${pgmout:-${jlogfile}}
+#touch $pgmout
+
+radmon_err_rpt=${radmon_err_rpt:-${USHradmon}/radmon_err_rpt.sh}
+base_file=${base_file:-$FIXgdas/gdas_radmon_base.tar}
+report=report.txt
+disclaimer=disclaimer.txt
+
+diag_report=diag_report.txt
+diag_hdr=diag_hdr.txt
+diag=diag.txt
+
+obs_err=obs_err.txt
+obs_hdr=obs_hdr.txt
+pen_err=pen_err.txt
+pen_hdr=pen_hdr.txt
+
+chan_err=chan_err.txt
+chan_hdr=chan_hdr.txt
+count_hdr=count_hdr.txt
+count_err=count_err.txt
+
+netcdf_boolean=".false."
+if [[ $RADMON_NETCDF -eq 1 ]]; then
+   netcdf_boolean=".true."
+fi
+
+DO_DATA_RPT=${DO_DATA_RPT:-1}
+RADMON_SUFFIX=${RADMON_SUFFIX:-opr}
+RAD_AREA=${RAD_AREA:-glb}
+REGIONAL_RR=${REGIONAL_RR:-0}
+rgnHH=${rgnHH:-}
+rgnTM=${rgnTM:-}
+SATYPE=${SATYPE:-}
+MAIL_TO=${MAIL_TO:-}
+MAIL_CC=${MAIL_CC:-}
+VERBOSE=${VERBOSE:-NO}
+LITTLE_ENDIAN=${LITTLE_ENDIAN:-0}
+USE_MAIL=${USE_MAIL:-1}
```

Comment:
USE_MAIL, MAIL_TO, and MAIL_CC can be removed.

Reply:
Removed